Symmetry-Adapted-Perturbation-Theory/SAPT0_no_S2.py

```diff
@@ -0,0 +1,264 @@
+# A script to compute the SAPT0 interaction energy without the the Single-Exchange Approximation
+# 
+# Created by: Jonathan M. Waldrop
+# Date: 1/9/2018
+# License: GPL v3.0
+# 
+# Relevant equations can be found in the following papers:
+# Jeziorski, B.; Bulski, M.; Piela, L. Int. J. Quantum Chem. 1976, 10, 281-297.
+# Schaffer, R.; Jansen, G. Theor. Chem. Acc. 2012, 131, 1235-1244.
+# Schaffer, R.; Jansen, G. Mol. Phys. 2013, 111, 2570-2584.
+
+import psi4,time
+import numpy as np
+np.set_printoptions(precision=5, linewidth=200, threshold=2000, suppress=True)
+from helper_SAPT import *
+
+# Set Psi4 & NumPy Memory Options
+psi4.set_memory('2 GB')
+psi4.core.set_output_file('sapt0_no_S2.dat', False)
+
+numpy_memory = 2
+
+# Set molecule to dimer
+dimer = psi4.geometry("""
+O   -0.066999140   0.000000000   1.494354740
+H    0.815734270   0.000000000   1.865866390
+H    0.068855100   0.000000000   0.539142770
+--
+O    0.062547750   0.000000000  -1.422632080
+H   -0.406965400  -0.760178410  -1.771744500
+H   -0.406965400   0.760178410  -1.771744500
+symmetry c1
+""")
+
+psi4.set_options({'basis': 'jun-cc-pVDZ',
+                  'e_convergence': 1e-8,
+                  'd_convergence': 1e-8})
+
+sapt = helper_SAPT(dimer, memory=8)
+
+### Overlap Matrix and Inverse
```

Comment:
Can you do this in multiple stages rather than a single line and draw a small diagram:

```S_{AA} S_{AB}
      S_{BA} S_{BB}```